build-scripts/vercel-build.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
```

Comment:
@Zequez we should probably deprecate this file now? I'm not sure if you're still using it, but running these processes in series will make the build far slower!